qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateAffiliationView.groovy

```diff
@@ -8,6 +8,7 @@ import com.vaadin.icons.VaadinIcons
 import com.vaadin.server.UserError
 import com.vaadin.shared.ui.ContentMode
 import com.vaadin.ui.*
+import com.vaadin.ui.themes.ValoTheme
```

Comment:
unnecessary import?

Reply:
I thought so too, but the buttons won't work if it's not specified